src/main.rs

```diff
@@ -1,4 +1,4 @@
-use std::collections::HashMap;
+use std::{collections::HashMap, path::Path};
```

Comment:
nit: should this import be part of line 2's import? IOW

```
use std::collections::HashMap;
use std::path::{Path, PathBuf};
```

Reply:
Good eye - thanks!  Fixed.